src/style-sheets/layout/_toolbar.scss

```diff
@@ -29,6 +29,7 @@
 
         &:hover {
           opacity: 0.7;
+          text-decoration: underline;
```

Comment:
Just need to remove the underlines for the links in the footer and toolbar and this PR should be good to go. Looks good, good work man

Reply:
ok I've removed these, so now only the green links (copy-link on hover) are underlined. I hope this is what you meant - wasn't sure if you wanted both the bottom-nav and the nav-link-list-container underlining removed, they are both in the footer so assumed so. 

cool, this is good experience as i've never used sass before, cheers.